velox/exec/Task.cpp

```diff
@@ -2761,11 +2758,21 @@ void Task::MemoryReclaimer::abort(
     return;
   }
   VELOX_CHECK_EQ(task->pool()->name(), pool->name());
```

Comment:
Consider to pass timeout in abort API  in followup to have end-to-end memory arbitration execution time control. Thanks!